JsonLd/ContextBuilder.php

```diff
@@ -119,7 +126,11 @@ public function getContext(ResourceInterface $resource = null)
             }
         }
 
-        return $context;
+        // Dispatch event to allow context customization
+        $event = new Event\ContextBuilderEvent($context);
```

Comment:
Based on [my comment](https://github.com/dunglas/DunglasApiBundle/pull/116#issuecomment-112859851), I think that the event should contain the base context and the resource object.